tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -532,7 +535,7 @@ class TestGetDagRunsEndDateFilters(TestDagRunEndpoint):
             (
                 f"api/v1/dags/TEST_DAG_ID/dagRuns?end_date_lte="
                 f"{(timezone.utcnow() + timedelta(days=1)).isoformat()}",
-                ["TEST_DAG_RUN_ID_1"],
+                ["TEST_DAG_RUN_ID_1", "TEST_DAG_RUN_ID_2"],
```

Comment:
This is because instead of changing 'running' to 'queued' for many affected tests, I decided to pass any given state to the second DagModel at line 135(_create_test_dag_run function). So when 'success' is passed to the function's state argument, then two dag_runs will have their end_date set.

Reply:
I have decided to just change from 'running' to 'queued' than adding this. Done